Content.Shared/Weapons/Misc/TetheringComponent.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Robust.Shared.GameStates;
+
+namespace Content.Shared.Weapons.Misc;
+
+/// <summary>
+/// Added to entities tethering something.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class TetheringComponent : Component
+{
+
+}
```

Comment:
```suggestion
public sealed partial class TetheringComponent : Component;
```
also rename this to something else that makes clear it's for the grappling gun, for example `GrapplingGunUserComponent`
this is too easy to confuse with the thether gun

Reply:
Donezo